btk/galsim_draw.py

```diff
@@ -0,0 +1,223 @@
+import galsim
```

Comment:
This whole piece of code comes from my multi-resolution simulations. It needs to be updated by me this week. and it's actually redundant with what's in draw_blends.